torch/nn/modules/activation.py

```diff
@@ -855,18 +855,21 @@ class Softshrink(Module):
         \end{cases}
 
     Args:
-        lambd: the :math:`\lambda` (must be no less than zero) value for the Softshrink formulation. Default: 0.5
+        lambd (float): The :math:`\lambda` (must be greater than or equal to zero) value for the Softshrink formulation. Default: 0.5
```

Comment:
The `float` annotation is already available on the official documentation. I believe it gets automatically generated
<img width="992" alt="Screenshot 2023-10-13 at 11 21 39 AM" src="https://github.com/pytorch/pytorch/assets/35276741/149bbae2-43cc-46fd-80c5-2a0d4a5d554c">